### ISSUE TITLE
Require students to set location via an interrupt screen directly after registration process

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#4556]
Chapters are reporting that students register and forget to complete location. Following the mentor and judge process, we would move the location screen to right after registration so they would need to select a location before accessing their dashboard. This will hopefully encourage more people to enter a location.

Talking with PT if we should target preview or QA for this change, depending on if we want it to take effect for this season or next.
![force location screen](https://github.com/Iridescent-CM/technovation-app/assets/516764/ef3292b1-eda5-4b95-9003-57ac0a3c0d0e)

**Comment by dboyer** [comment]
Confirmed that PT would like this on production now, rather than waiting until next season.

**Comment by shaun-technovation** [comment]
This is now on QA.

**Comment by dboyer** [comment]
Tested on QA and confirmed:

- Could register a new junior division student account and saw the location screen right after registration before I got to my dashboard. I was able to request parental consent/media consent forms and everything worked as expected.
- Was then able to change location for that same account using the location tab on the student dashboard
- Registered as another new student and got to the location screen. Tried to click on the dashboard, profile, and scores tab and was redirected to the location interrupt. Closed the tab and then went back to the site. It returned me to the location screen again. Filled in the location and was able to go to my dashboard and profile.
- I logged in as a student with an existing account but no location selected. I was taken to the location screen and was not able to get past it to my dashboard until I entered a location.

Kiki also tested and everything worked as expected.
